articles/architecture-scenarios/application/spa-api.md

```diff
@@ -2,27 +2,31 @@
 order: 01
 title: SPA + API
 image: /media/articles/architecture-scenarios/spa-api.png
-extract: Single Page Web Application “Client” which talks to an API (“Resource Server”). The application will use OpenID Connect with the Implicit Grant Flow to authenticate users with Auth0.
+extract: Single Page Web Application "Client" which talks to an API ("Resource Server"). The application will use OpenID Connect with the Implicit Grant Flow to authenticate users with Auth0.
 description: Explains the architecture scenario with a Single Page Web Application to an API using OpenID Connect with the Implicit Grant Flow to authenticate users with Auth0.
 ---
 
 # SPA + API
 
-![](/media/articles/architecture-scenarios/spa-api.png)
+![SPA + API Flow](/media/articles/architecture-scenarios/spa-api.png)
 
-In this scenario you have a Single Page Web Application “Client” which talks to an API (“Resource Server”). The application will use OpenID Connect with the Implicit Grant Flow to authenticate users with Auth0.
+In this scenario you have a Single Page Web Application ("Client") which talks to an API ("Resource Server"). The application will use **OpenID Connect** with the **Implicit Grant Flow** to authenticate users with Auth0. Note that this flow can only be used for Clients whose type is **Single Page Application** in the [Dashboard](${manage_url}).
 
-When a user logs in, Auth0 will return to the application an `access_token` as well as an `id_token`. The `id_token` is used to securely call the API on behalf of the user. Alternatively the user profile can be obtained by calling the `/userinfo` endpoint in the Auth0 Authentication API with the `access_token`.
+When a user logs in, Auth0 will return to the application an `access_token` and optionally an `id_token`:
 
-The application will usually store the information about the user’s session (i.e. whether they are logged in, their tokens, user profile data, etc) inside some sort of storage such a Local Storage.
+- The `access_token` is used to securely call the API on behalf of the user.
+
+- The `id_token` is consumed only by the client and contains user profile data. Alternatively the user profile can be obtained by calling the `/userinfo` endpoint in the Auth0 Authentication API with the `access_token`.
```

Comment:
Need to mention that `/userinfo` is only callable if `openid` was granted as a scope.